File: Source/UserManagement/Web.Angular/src/app/domain/data-collector/ChangeLocation.ts
import { Command } from '../../services/Command';
import {Location} from '../location.model'
export class ChangeLocation extends Command {
    dataCollectorId: string;
    location: Location

    constructor() {
        super();
        this.type = 'CBS#UserManagement.DataCollector.Changing-ChangeLocation+Command@1';
        this.location = new Location();
    }
}

File: Source/UserManagement/Domain/DataCollectors/Changing/ChangeLocation.cs
using System;
using Concepts;
using Concepts.DataCollector;
using Dolittle.Commands;

namespace Domain.DataCollectors.Changing
{
    public class ChangeLocation : ICommand
    {
        public DataCollectorId DataCollectorId { get; set; }
        public Location Location { get; set; }
    }
}